vllm_ascend/envs.py

```diff
@@ -159,6 +159,11 @@
     #   1: enable moe all2all seq.
     "VLLM_ASCEND_ENABLE_MOE_ALL2ALL_SEQ":
     lambda: bool(int(os.getenv('VLLM_ASCEND_ENABLE_MOE_ALL2ALL_SEQ', '0'))),
+    # Whether to enable layernorm.py torch_npu.npu_add_rms_norm_quant
+    # 0: default
+    # 1: enable
+    "USE_ADD_RMSNORM_QUANT":
```

Comment:
Why is this switch necessary when using quantitative model weights?

In other words, are there any disadvantages to using `AddRMSNormW8A8Quant`?